data/sql/updates/pending_db_world/rev_1643994847133627900.sql

```diff
@@ -0,0 +1,1832 @@
+(3112,"zhTW","願那些不潔、骯髒的野蠻人受到審判。讚揚聖光!$B$B讓我們的兄弟姊妹知道這個奇蹟，大將軍。$B$B願聖光永遠地保護你，讓你可以傳播那些善言直到世界終結。$B$B滿懷敬意的，$B$B丹·亞當斯$B結尾",0),
+(3113,"zhTW","1. 留出一蓬鬍鬚，因為這可以掩飾面對聖光時的不完美。$B$B2. 以母馬的馬奶潔淨你的雙手，在奉獻時的雙手應該柔軟而滑順。$B$B3. 在河中沐浴，讓聖光榮耀的自然禮讚清潔你的皮膚。",0),
+(3114,"zhTW","4. 換下你日常的聖袍，讓你的外觀和氣味不會觸怒聖光的僕人。$B$B5. 隨時隨地都維持挺直的姿勢 - 不要讓疲勞或懶散使你的脊椎彎下。$B$B<清單後續還有好幾頁的注意事項>",0),
+(3115,"zhTW","從高階修士蘭格瑞的桌上:$B$B在黑暗時刻的混亂與鬥爭中，一個人好像應該轉向內，保護自己的家人和所愛，排除其他的。但這種想法是錯的 - 沒有教會這樣的希望燈塔，絕望會在你家人的靈魂中氾濫。喫飽、穿暖和快樂的笑容，這些世俗好事的追求無法拯救你家人心中的空虛，勸你不要逃避對聖光的職責。",0),
+(3116,"zhTW","交付給教會。在世俗享受匱乏時的艱苦忍耐就像是靈魂的熔爐，能夠燒去一切的不純淨，將你提升到更高的層次。不要因為肚子餓得咕咕叫，赤著腳被尖石割傷，或因為寒風在顫抖而感到羞愧。這只是聖光對靈魂的激勵與考驗。這些犧牲確保那些獻給聖光高入穹頂、映著金光的頌歌能夠激勵你的信仰。$B$B棄絕肉體的享受，而聖光就靠你越近。",0),
+(3119,"zhTW","靠勞動來淨化身體。靠祈禱來淨化靈魂。靠火焰來淨化敵人。疼痛不是你的敵人，而是值得擁抱的盟友，緊緊地擁抱它，傳播給不信者與信者。$B$B抵抗聖光的道路是不潔的象徵，應該要將他的血肉割下。",0),
+(3117,"zhTW","放縱沉溺的主題是很困難的，但在我們的生活中，這是必要之惡。小小的背離正義，脫口而出的粗話，憤怒時的拳打腳踢 - 這些所有事情都會使一個人衰弱，而教會必需要確認這一切。$B$B然而，這樣的確認並非毫無代價。粗話的罪行可能只需要一些銀幣就能淨化你的靈魂。但是暴力的罪行就需要金幣纔行。至於其他的罪行，完全的坦承認罪，和慷慨的奉獻都是必須的。",0),
+(3118,"zhTW","毆打:$B毆打臉部: 2金幣$B毆打下體: 3金幣$B毆打胸膛: 1金幣45銀幣$B $B<一份關於罪刑與價格，長達好幾頁的詳細列表。>",0),
+(3120,"zhTW","大將軍，$B$B壁爐谷和提里斯法的軍隊只離新亞法隆不到一天的距離。我們飛馳而來，聖光在我們的背上閃耀，風在我們的馬蹄間流轉。很快地天譴軍團就得對付血色十字軍的全部力量!$B$B聖光祝福你，$B$B高階指揮官嘉爾瓦·波爾伯勒",0),
+(3121,"zhTW","<HTML>$B<BODY>$B$B$B<H1 align=\"center\">贖罪之途",0),
+(3279,"zhTW","奴僕們聽令!巫妖王將讓德拉庫魯指揮在祖爾德拉克的天譴軍部隊。我們的命令是殺掉所有東西。$B$B所有東西都得死!$B$B我們將走在亡者原野的屍體上。為了天譴軍，將他們送到在賽拉瑪斯的納法勒斯王子部隊!在德拉庫魯的管理下，所有東西都不會浪費!很快，我們就會喝著先知的血，變得比以往都要強大!祖爾德拉克跟剛德拉克都將死於天譴軍手上!",0),
+(3535,"zhTW","大領主，$B$B$N成功地帶領我們對抗來自天譴岸地的天譴軍，並使我們得以開始向寒冰皇冠進軍。$B$B泥瓦匠帶滿了足以興建正義要塞的材料，這都多虧了$N和$g他:她;在水晶之歌森林的戰功。$B$B還剩下最後一個挑戰-十字軍之巔。我希望從這個最讓我們苦惱的所在，開始劃下這個時代的終章。$B$B我們等待著您的命令，大人。$B$B願聖光保護您。$B$B-賈斯塔夫",0),
+(3536,"zhTW","我的主上，$B$B慈悲的王，願聖光令您刀下留人，並平復您的心跳。請暫緩您對這個死去勇士的判決 - 即使一下子也好 - 並請聆聽我的話語。$B$B帶著這封信的使者生前曾是聯盟的英雄。雖然$g他:她;散發著死亡的惡臭與天譴軍的外表，$g他:她;卻擁有勇士的靈魂。一個適才返回軀體之中的靈魂。",0);
+INSERT INTO `page_text_locale` VALUES
+(3537,"zhTW","$N代表一支全新團結對抗巫妖王的力量，人稱黯刃騎士。沒錯，我的主上，死亡騎士們轉而對抗他們的前任主人。大領主亞歷山卓斯·莫格萊尼 - 『灰燼使者』- 的兒子領導著這個組織。他們的目標和我們一致:消滅天譴軍團。$B$B我所請求的不是讓$N與黯刃騎士加入聯盟，而僅僅是您的寬恕與容忍而已。",0),
+(3538,"zhTW","記住，殿下，為了鮮血與榮耀。$B$B您恭敬的，$B$B大領主提裏奧·弗丁",0),
+(3539,"zhTW","大酋長，$B$B偉大的戰帥，願聖光令您刀下留人，並平復您的心跳。請暫緩您對這個死去勇士的判決 - 即使一下子也好 - 並請聆聽我的話語。$B$B帶著這封信的使者生前曾是部落的英雄。雖然$g他:她;散發著死亡的惡臭與天譴軍的外表，$g他:她;卻擁有勇士的靈魂。一個適才返回軀體之中的靈魂。",0),
+(3540,"zhTW","$N代表一支全新團結對抗巫妖王的力量，人稱黯刃騎士。沒錯，我的主上，死亡騎士們轉而對抗他們的前任主人。大領主亞歷山卓斯·莫格萊尼 - 『灰燼使者』- 的兒子領導著這個組織。他們的目標和我們一致:消滅天譴軍團。$B$B我所請求的不是讓$N與黯刃騎士加入部落，而僅僅是您的寬恕與容忍而已。",0),
+(3541,"zhTW","記住，大酋長，我們都是為了鮮血與榮耀。$B$B您恭敬的，$B$B大領主提裏奧·弗丁$B$BP.S替我向伊崔格打聲招呼，並告訴他我在北裂境需要一個能幹的獸人 - 當然，在得到您的允許後。",0),
+(3543,"zhTW","年復一年，我被關在這個魔法監獄中。今晚，我就要潛逃出去。我備齊了脫逃的一切所需。$B$B<墨水浸水暈開，模糊了信件的這個部分>$B$B為什麼我得在這個鬼地方這麼久?我到底做了什麼才受此懲罰?日復一日，這些問題在我腦海中打轉。為什麼有人會落得如此下場?$B$B<墨水浸水暈開，模糊了信件的這個部分>$B$B也許我會在來生瞭解這個答案。",0),
+(3040,"zhTW","<HTML>$B<BODY>$B$B$B<P>好吧，$N，這張清單列出了我們最新的超級機密，呃…超級武器所需要的一切東西。目前，就先叫它「卡芙緹的超進階初體版微縮光束槍」吧。",0),
+(3041,"zhTW","<HTML>$B<BODY>$B$B$B<H1 align=\"center\">我的工具",0),
```

Comment:
There are plenty rows with this kind of html stuff.

Reply:
some text actually use html